### PR TITLE
BLS coverage result has a different name than the rest

### PR DESCRIPTION
**Pull request by vncoelho**
close https://github.com/neo-project/neo/issues/3160

### CODE REVIEW

**Comment by vncoelho** [comment]
locally it fixed

```yaml
#21 19.95 +----------------------------+--------+--------+--------+
#21 19.95 | Module                     | Line   | Branch | Method |
#21 19.95 +----------------------------+--------+--------+--------+
#21 19.95 | Neo.Cryptography.BLS12_381 | 83.22% | 55%    | 69.2%  |
#21 19.95 +----------------------------+--------+--------+--------+
#21 19.95 | Neo.ConsoleService         | 15.66% | 13.04% | 19.4%  |
#21 19.95 +----------------------------+--------+--------+--------+
#21 19.95 
#21 19.95 +---------+--------+--------+--------+
#21 19.95 |         | Line   | Branch | Method |
#21 19.95 +---------+--------+--------+--------+
#21 19.95 | Total   | 68.29% | 30.67% | 60.47% |
#21 19.95 +---------+--------+--------+--------+
#21 19.95 | Average | 49.44% | 34.01% | 44.3%  |
#21 19.95 +---------+--------+--------+--------+
```

then 
```yaml
#23 26.94 +----------------------------+--------+--------+--------+
#23 26.94 | Module                     | Line   | Branch | Method |
#23 26.94 +----------------------------+--------+--------+--------+
#23 26.94 | Neo.Cryptography.BLS12_381 | 87.99% | 63%    | 73.96% |
#23 26.94 +----------------------------+--------+--------+--------+
#23 26.94 | Neo.IO                     | 75.36% | 83.82% | 75%    |
#23 26.94 +----------------------------+--------+--------+--------+
#23 26.94 | Neo.VM                     | 33.95% | 25.9%  | 41.88% |
#23 26.94 +----------------------------+--------+--------+--------+
#23 26.94 | Neo                        | 78.88% | 65.32% | 81.52% |
#23 26.94 +----------------------------+--------+--------+--------+
#23 26.94 | Neo.Extensions             | 78.57% | 0%     | 80%    |
#23 26.94 +----------------------------+--------+--------+--------+
#23 26.94 | Neo.Json                   | 36.23% | 13.66% | 40.46% |
#23 26.94 +----------------------------+--------+--------+--------+
#23 26.94 | Neo.ConsoleService         | 15.66% | 13.04% | 19.4%  |
#23 26.94 +----------------------------+--------+--------+--------+
```

**Comment by vncoelho** [comment]
`coverage/coveralls — Coverage increased (+1.9%) to 75.829% `

**Review by Jim8y — DISMISSED** [review]

**Comment by Jim8y** [comment]
LGTM

**Comment by Jim8y** [comment]
@shargon up to you.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -27,14 +27,17 @@ jobs:
...
+      if: matrix.os == 'windows-latest' || matrix.os == 'macos-latest'
+      run: |
+        dotnet test
+    - name: Test for coverall
       if: matrix.os == 'ubuntu-latest'
       run: |
         find tests -name *.csproj | xargs -I % dotnet add % package coverlet.msbuild
-    - name: Test
-      run: |
         dotnet test ./tests/Neo.ConsoleService.Tests         /p:CollectCoverage=true /p:CoverletOutput=${GITHUB_WORKSPACE}/coverage-join/
```
[the remark this answers is not in the document]
Comment (vncoelho) [reply .github/workflows/main.yml:38]: Just BLS

**Comment by vncoelho** [comment]
> There is something wrong, we should do it or after the first, or in all of them, we need to check

You can do it first and avoid the mv

**Comment by vncoelho** [comment]
like the last commit, take a look @shargon 

let's see if the result will be the same

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -27,14 +27,16 @@ jobs:
       if: matrix.os == 'ubuntu-latest'
       run: |
          dotnet format --verify-no-changes --verbosity diagnostic
-    - name: Prepare coverage
+    - name: Test
+      if: matrix.os == 'windows-latest' || matrix.os == 'macos-latest'
```
Comment (shargon) [inline .github/workflows/main.yml:31]:
```suggestion
      if: matrix.os != 'ubuntu-latest'
```

**Review by shargon — APPROVED** [review]

**Pull request merged**